applications/ContactStructuralMechanicsApplication/custom_processes/master_slave_process.cpp

```diff
@@ -35,48 +35,42 @@ void MasterSlaveProcess::Execute()
     
     // Creating a buffer for parallel vector fill
     const int num_threads = OpenMPUtils::GetNumThreads();
-    std::vector<std::size_t> index_node;
-    std::vector<std::vector<std::size_t>> index_node_buffer(num_threads);
-    std::vector<std::size_t> index_cond;
-    std::vector<std::vector<std::size_t>> index_cond_buffer(num_threads);
+    std::vector<IndexType> index_node;
+    std::vector<std::vector<IndexType>> index_node_buffer(num_threads);
+    std::vector<IndexType> index_cond;
+    std::vector<std::vector<IndexType>> index_cond_buffer(num_threads);
     
     #pragma omp parallel
     {
         const int thread_id = OpenMPUtils::ThisThread();
```

Comment:
it would be best if you avoid using the thread_id, to be future proof. Do you really need it?

Reply:
Yes, to access to each thread vector